x-pack/plugins/security_solution/public/detections/pages/detection_engine/rules/translations.ts

```diff
@@ -992,3 +898,226 @@ export const SHOWING_EXCEPTION_LISTS = (totalLists: number) =>
+  }
+);
+
+export const RULES_BULK_DELETE_FAILURE_DESCRIPTION = (rulesCount: number) =>
+  i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.delete.errorToastDescription',
+    {
+      values: { rulesCount },
+      defaultMessage: '{rulesCount, plural, =1 {# rule is} other {# rules are}} failed to delete.',
+    }
+  );
+
+/**
+ * Bulk Enable
+ */
+
+export const RULES_BULK_ENABLE_SUCCESS = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.enable.successToastTitle',
+  {
+    defaultMessage: 'Rules enabled',
+  }
+);
+
+export const RULES_BULK_ENABLE_SUCCESS_DESCRIPTION = (totalRules: number) =>
+  i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkAction.enable.successToastDescription',
+    {
+      values: { totalRules },
+      defaultMessage:
+        'Successfully enabled {totalRules, plural, =1 {{totalRules} rule} other {{totalRules} rules}}',
+    }
+  );
+
+export const RULES_BULK_ENABLE_FAILURE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.enable.errorToastTitle',
+  {
+    defaultMessage: 'Error enabling rules',
+  }
+);
+
+export const RULES_BULK_ENABLE_FAILURE_DESCRIPTION = (rulesCount: number) =>
+  i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.enable.errorToastDescription',
+    {
+      values: { rulesCount },
+      defaultMessage: '{rulesCount, plural, =1 {# rule is} other {# rules are}} failed to enable.',
+    }
+  );
+
+/**
+ * Bulk Disable
+ */
+
+export const RULES_BULK_DISABLE_SUCCESS = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.disable.successToastTitle',
+  {
+    defaultMessage: 'Rules disabled',
+  }
+);
+
+export const RULES_BULK_DISABLE_SUCCESS_DESCRIPTION = (totalRules: number) =>
+  i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.disable.successToastDescription',
+    {
+      values: { totalRules },
+      defaultMessage:
+        'Successfully disabled {totalRules, plural, =1 {{totalRules} rule} other {{totalRules} rules}}',
+    }
+  );
+
+export const RULES_BULK_DISABLE_FAILURE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.disable.errorToastTitle',
+  {
+    defaultMessage: 'Error disabling rules',
+  }
+);
+
+export const RULES_BULK_DISABLE_FAILURE_DESCRIPTION = (rulesCount: number) =>
+  i18n.translate(
+    'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.disable.errorToastDescription',
+    {
+      values: { rulesCount },
+      defaultMessage: '{rulesCount, plural, =1 {# rule is} other {# rules are}} failed to disable.',
+    }
+  );
+
+/**
+ * Bulk Edit
+ */
+
+export const RULES_BULK_EDIT_SUCCESS = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.allRules.bulkActions.edit.successToastTitle',
+  {
+    defaultMessage: 'Rules changes updated',
```

Comment:
Perhaps `Rules updated` for success toast title (matches other success actions: `Rules exported`, `Rules duplicated`, etc)
<img width="349" alt="image" src="https://user-images.githubusercontent.com/2946766/159617929-37aa2725-9d11-4ba6-83b7-9c26f32d6247.png">

```suggestion
    defaultMessage: 'Rules updated',
```